org.gnome.Hamster.yaml

```diff
@@ -0,0 +1,54 @@
+---
+app-id: org.gnome.Hamster
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: hamster
+finish-args:
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--share=ipc'
+  - '--filesystem=xdg-documents'
+modules:
+  - name: intltool
```

Comment:
Please switch to intltool from https://github.com/flathub/shared-modules/tree/master/intltool.